src/update-manager.ts

```diff
@@ -113,15 +151,22 @@ export default class UpdateManager {
 
     var request
     var definitive = false
+    const meta = this.store.fetcher.appNode
+    // const kb = s
+
      // @ts-ignore passes a string to kb.each, which expects a term. Should this work?
-    var requests = kb.each(undefined, this.ns.link('requestedURI'), docpart(uri))
+    var requests = kb.each(undefined, this.ns.link('requestedURI'), docpart(uri), meta)
     var method: string
     for (var r = 0; r < requests.length; r++) {
       request = requests[r]
       if (request !== undefined) {
-        var response = kb.any(request, this.ns.link('response')) as Quad_Subject
-        if (request !== undefined) {
-          var wacAllow = kb.anyValue(response, this.ns.httph('wac-allow'))
+        const response = kb.any(request, this.ns.link('response'), null, meta) as Quad_Subject
```

Comment:
@timbl now that we are looking up the response meta data in the `meta` graph we need to make sure that the fetcher stores it there as well. Currently the fetcher `saveResponseMetadata` uses a blank node and I think it needs to use `this.appNode` instead. This is the reason why the tests in `update-manager-sparql-insert-test.ts` are currently failing

Reply:
@bourgeoa why did you resolve this? If my assumption is wrong, I would like to understand why.

Reply:
ok. But this still puzzles me. The line will find `any` response. There might be an `outOfDate` one and a current one in the store. It might just pick the `outOfDate` one and `editable` will return undefined then, despite having a more up to date response that says it is editable via SPARQL. I guess we need to find any response that is not out of date.

Reply:
From the call: all good, we are looping through the requests!